### PR TITLE
Replaced all the css into tailwind css on Card component

### PR DESCRIPTION
**Pull request by subhajit20**
# Description of changes

Replaced all the css into tailwind css on Card component
Issue - #1725 

Before:
Before

After:
After

# Issue Resolved

Fixes #NA

### CODE REVIEW

File: components/Cards/Card/Card.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import { bool, node, string } from 'prop-types';
 import classNames from 'classnames';
 import { getDataAttributes } from 'common/utils/prop-utils';
-import styles from './Card.module.css';
```
Comment (kylemh) [inline components/Cards/Card/Card.js:4]: please delete the file 😄

Comment (subhajit20) [reply components/Cards/Card/Card.js:4]: every time I forgot to delete this file

Comment (subhajit20) [reply components/Cards/Card/Card.js:4]: Done

File: components/Cards/Card/Card.js [hunk 2]
```diff
@@ -19,7 +18,14 @@ function Card({ children, className, hasAnimationOnHover, ...props }) {
 
   return (
     <article
-      className={classNames(styles.Card, className, { [styles.animatedCard]: hasAnimationOnHover })}
+      className={classNames(
+        'items-center bg-white text-secondary flex flex-col flex-nowrap justify-around m-4 min-h-[100px] min-w-[100px] p-6 shadow-md focus-visible:outline-none',
```
Comment (kylemh) [inline components/Cards/Card/Card.js:22]: I'm not sure of the specific fill value, but we just need to add `[&_svg]:fill-WHATEVER` to finish this one

**Comment by subhajit20** [comment]
I am deleting this branch as I am facing some issue and creating new pr.

**Pull request closed without merging**